css/style.css

```diff
@@ -0,0 +1,19 @@
+.nav {
+  background-color: #be3144;
+}
+
+#welcome-section {
+  height: 100vh;
```

Comment:
Please change the height of the Welcome section. Your welcome should have the dimensions as in freeCodeCamp's responsive web design course project.